packages/flutter_test/lib/src/window.dart

```diff
@@ -0,0 +1,246 @@
+import 'dart:typed_data' show ByteData;
+import 'dart:ui' hide window;
+
+import 'package:meta/meta.dart';
+
+/// [Window] that wraps another [Window] and allows faking of some properties
+/// for testing purposes.
```

Comment:
we should elaborate some more here. Maybe some sample code, talk about what happens to overridden values when one test ends and another starts, talk about the general approach (overriding, null values allowing passthrough, etc)...

Reply:
Done